### PR TITLE
Add papers.

### PR DESCRIPTION
**Pull request by Pavel-Durov**
Added more papers.
Not sure if all of them are relevant for general use :)

### CODE REVIEW

File: softdev.bib [hunk 1]
```diff
@@ -3335,19 +3335,36 @@ @article{yanovski21ghostcell
...
-    booktitle = {DLS}
+  title = "Efficient Interpretation using Quickening",
+  author = "Brunthaler, Stefan",
+  year = 2010,
+  month = Oct,
+  doi = {10.1145/1869631.1869633},
+  booktitle = {DLS}
+}
+
+@inproceedings{bolz07not,
```
Comment (Pavel-Durov) [inline softdev.bib:3355]: could not find `booktitle` for this one...

File: softdev.bib [hunk 2]
```diff
@@ -3335,19 +3335,36 @@ @article{yanovski21ghostcell
 }
 
 @inproceedings{cheng20typefreezing,
```
Comment (ltratt) [inline softdev.bib:3337]: This one and the next seem to just be changing whitespace?

Comment (Pavel-Durov) [reply softdev.bib:3337]: yes I re-formatted it to have the same indentation :)

Comment (ltratt) [reply softdev.bib:3337]: In the future, I don't mind that being in another commit, but the problem with whitespace changes is that they detract from the more important bits.

File: softdev.bib (hunk 1 above)
Comment (ltratt) [reply softdev.bib:3355]: "3rd Workshop on Dynamic Languages and Applications" according to https://scholar.google.com/citations?view_op=view_citation&hl=en&user=S0rpYpkAAAAJ&citation_for_view=S0rpYpkAAAAJ:8k81kl-MbHgC

File: softdev.bib (hunk 2 above)
Comment (Pavel-Durov) [reply softdev.bib:3337]: got it.

File: softdev.bib (hunk 1 above)
Comment (Pavel-Durov) [reply softdev.bib:3355]: Ah, I thought we keep `booktitle` for events/conferences. 
Are journals basically the same thing?

Comment (ltratt) [reply softdev.bib:3355]: From memory, we use 'booktitle' to mean "event" (you can look elsewhere in the file for precedent).

Comment (Pavel-Durov) [reply softdev.bib:3355]: Updated :point_right: 6770bc91682d12daabe142fb83793807ea26c05d

**Comment by ltratt** [comment]
Please

File: softdev.bib [hunk 3]
```diff
@@ -3335,19 +3335,37 @@ @article{yanovski21ghostcell
...
+@inproceedings{bolz07not,
+  title = {How to not write virtual machines for dynamic languages},
+  author = {Bolz, Carl Friedrich and Rigo, Armin},
+  year = {2007},
+  booktitle = {DLA},
+  month = July
+}
+
+@inproceedings{ancona07rpython,
+  title = {RPython: a step towards reconciling dynamically and statically typed OO languages},
```
Comment (ltratt) [inline softdev.bib:3364]: Just spotted: you'll need to add curly brackets around "RPython" and "OO".

**Comment by ltratt** [comment]
Please squash.

**Comment by Pavel-Durov** [comment]
Squashed :point_right: 6916d3d0ee795e6ad1dd294b9716f5e5bd63d718

**Comment by Pavel-Durov** [comment]
Build failed with this error:
```
***warn: Unknown month string: 'July'
```
fixed in :point_right: 274ff1088c92112b4d2a341dbaa1402548e5e523

**Comment by ltratt** [comment]
Please squash.

**Comment by Pavel-Durov** [comment]
squashed :point_right: f80d2dc4c4dfc4c0d2c8cb5237eb477939c59f7d

**Pull request merged**